iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/jproxy/impl/ProxyNegotiationHandler.java

```diff
@@ -175,6 +174,8 @@ private void copyBytesBetweenClientAndService(final ReadWriteState readWriteStat
             final Scanner requestScanner = new Scanner(request);
             final String firstLine = requestScanner.nextLine();
 
+            System.out.println(request);
```

Comment:
Please use `log`

Reply:
I'll remove this print, it was added by mistake